docs/polymorphism.rst

```diff
@@ -0,0 +1,36 @@
+Polymorphism
+============
+
+PynamoDB supports polymorphism through the use of discriminators.
+
+A discriminator is a value that is written to DynamoDB that identifies the python class being stored.
+(Note: currently discriminators are only supported on MapAttribute subclasses; support for model subclasses coming soon.)
+
+Discriminator Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The discriminator value is stored using a special attribute, the DiscriminatorAttribute.
+Only a single DiscriminatorAttribute can be defined on a class.
+
+The discriminator value can be assigned to a class as part of the definition:
+
+.. code-block:: python
+
+    class ParentClass(MapAttribute):
+        _cls = DiscriminatorAttribute()
```

Comment:
Should we make any recommendation or note about the attribute name? Underscore-prefixed is unusual for these

Reply:
habit from mongo -- will change it to `cls`